models/Profile.js

```diff
@@ -69,11 +70,20 @@ const ProfileSchema = new mongoose.Schema(
         price: {
           startingFrom: Number,
         },
+        location:{
+          road:String,
```

Comment:
combine road, city,state, & country into one value.
Our map positioning is approximate(+/- 1km), so there is no reason for highly accurate location info.

Reply:
Should I keep only city ? Because we are fetching the coordinates from only city variable.

Reply:
I just looked up the docs for events.
I didn't realize this was part of the spec for events, but wasn't stored in the database.
We should keep the fields as described in the docs, but for fetching the coordinates, you need to combine city, state & country.

Reply:
We are only fetching the coordinates from city name so i think there is no need to combine all these.

Reply:
You need to fetch the location using all of them.

If you had an event in Manchester, NH, USA and only sent Manchester to the location API, it would likely return the coordinates of Manchester, England

Reply:
Done in new commit.